.circleci/workflows.yml

```diff
@@ -0,0 +1,2898 @@
+version: 2.1
```

Comment:
This file is 1:1 with the previous `config.yml` file. CircelCI uses `config.yml` by default. This change is to simple limit kicking off the workflows. Over time,  plan to make enhancements to the dynamic configuration to simplify this setup.